lib/experimental/Layouts/ProfileLayout/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { forwardRef } from "react"
+
+interface ProfileLayoutProps {
+  mainContent: React.ReactNode
+  sideContent: React.ReactNode
+}
+
+export const ProfileLayout = forwardRef<HTMLDivElement, ProfileLayoutProps>(
+  ({ mainContent, sideContent }, ref) => {
+    return (
+      <div
+        ref={ref}
+        className="grid min-h-svh grid-cols-1 py-6 sm:grid-cols-[7fr_3fr] sm:divide-x sm:divide-y-0 sm:divide-solid sm:divide-f1-foreground-disabled"
```

Comment:
I think grid makes sense precisely because of that "reordering" property! It could of course be solved reversing the order of flex as well but grid feels more natural.